scripts/ignore_translation.toml

```diff
@@ -0,0 +1,16 @@
+[nl_NL]
+ignore = [
+    "language.direction",
+    "error",
+    "help",
+    "sponsor",
+    "info",
+    "navbar.allTools",
+    "showJS.downloadJS",
+    "getPdfInfo.downloadJson",
+    "HTMLToPDF.print",
+    "HTMLToPDF.screen",
```

Comment:
Why ignore screen? I see you editted it

Reply:
Oh, sorry must have added that before I realized it could be translated.